source/common/http/conn_manager_impl.cc

```diff
@@ -2881,17 +2914,25 @@ void ConnectionManagerImpl::ActiveStreamEncoderFilter::responseDataTooLarge() {
   }
 }
 
+void ConnectionManagerImpl::ActiveStream::onResponseDataTooLarge() {
+  connection_manager_.stats_.named_.rs_too_large_.inc();
```

Comment:
Do we have a TODO for how to more cleanly handles stats client and server side?
seems like the manager could do stats if it had pointers to them.

Reply:
No TODO yet, I'll add one. I'm a bit unsure whether this should be callbacks, required counters or optional counters (if we require counters we'll need to add stats for this for upstream calls, which would add more default stats etc.)